### PR TITLE
Added bolt11 QR to payment screen.

### PR DESCRIPTION
**Pull request by niteshbalusu11**
Signed-off-by: Nitesh Balusu <niteshbalusu@icloud.com>

Added qr code to payments screen.
Added a dependency qrcode.react.

### CODE REVIEW

**Comment by niteshbalusu11** [comment]
Screen Shot 2023-01-24 at 11 07 55 AM

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
```
Comment (bnonni) [inline .vscode/settings.json:1]: Hey, can you delete this from the PR and .gitignore it?

Comment (niteshbalusu11) [reply .vscode/settings.json:1]: The stuff that got added to the settings file by me? I could just make another commit removing those.

Comment (bnonni) [reply .vscode/settings.json:1]: Ahh, i did not realize this was already there. Then nvm. You can leave it.

**Review by bnonni — APPROVED** [review]
LGTM!

**Comment by bnonni** [comment]
@niteshbalusu11 can you clone the origin repo, create this branch there and push? For whatever reason, the checks wont run on this PR, but when I created a duplicate branch in the local repo, they ran just fine.

**Review by bnonni — APPROVED** [review]

**Review by sbddesign — APPROVED** [review]
YOLO

**Pull request merged**